Formula/gh.rb

```diff
@@ -0,0 +1,25 @@
+class Gh < Formula
+  desc "The GitHub CLI tool"
```

Comment:
```suggestion
  desc "GitHub command-line tool"
```

Probably better without an article.